hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/token/BlockTokenVerifier.java

```diff
@@ -71,8 +71,8 @@ protected OzoneBlockTokenIdentifier createTokenIdentifier() {
   @Override
   protected Object getService(ContainerCommandRequestProtoOrBuilder cmd) {
     BlockID blockID = HddsUtils.getBlockID(cmd);
-    Preconditions.checkNotNull(blockID,
-        "no blockID in %s command", cmd.getCmdType());
+    Objects.requireNonNull(blockID,
+        String.format("no blockID in %s command", cmd.getCmdType()));
```

Comment:
Message should be lazily produced.

```suggestion
    Objects.requireNonNull(blockID, () -> "blockID == null in command " + cmd.getCmdType());
```